cloudwatchmetrics/src/main/java/software/amazon/awssdk/metrics/internal/cloudwatch/MetricUploaderThread.java

```diff
@@ -53,7 +53,7 @@ public void run() {
                 Iterable<PutMetricDataRequest> requests = qIterator.nextUploadUnits();
                 for (PutMetricDataRequest req : requests) {
                     appendUserAgent(req);
-                    log.debug(req);
+                    log.debug(req.toString());
```

Comment:
Will switch to log.debug("{}", req) which should prevent formatting unless debug is enabled